sips/sip-merge-staked-sui.md

```diff
@@ -0,0 +1,133 @@
+| SIP-Number          | 31 |
+| ---:                | :--- |
+| Title               | Fungible StakedSui objects |
+| Description         | Allow StakedSui objects to be transformed into a FungibleStake object, which is epoch independent. |
+| Author              | ripleys <0xripleys@solend.fi> |
+| Editor              | Amogh Gupta <amogh@sui.io, @amogh-sui>  |
+| Type                | Standard |
+| Category            | Framework |
+| Created             | 2024-05-23 |
+| Comments-URI        | https://sips.sui.io/comments-31 |
+| Status              | Draft |
+| Requires            | NA |
+
+## Abstract
+
+Allow StakedSui objects to be transformed into an Fungible object, which are fungible and epoch independent. This improves efficiency for anyone working with a large amount of StakedSui objects, primarily liquid staking derivative contracts.
+
+## Motivation
+
+Currently, StakedSui objects are tied to a specific activation epoch. If two StakedSui objects are from different activation epochs, they cannot be merged together. 
+
+This is annoying for anyone working with a large amount of StakedSui objects, primarily liquid staking derivative (LST) contracts. Currently, any LST contract now has to manage O(n) StakedSui objects _per_ validator, where n is the number of epochs since genesis. See [the vSui implementation](https://github.com/Sui-Volo/volo-liquid-staking-contracts/blob/main/liquid_staking/sources/validator_set.move#L45) for reference. This is inefficient, as any LST unstake operation can now touch a large amount of StakedSui objects.
+
+## Specification
+
+[staking_pool.move](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/packages/sui-system/sources/staking_pool.move)
+
+```move
+
+    /// An FungibleStake object with a value of 1 corresponds to 1 pool token in the Staking pool.
+    /// This can be a Coin! See the Rationale below.
+    public struct FungibleStake has key, store {
```

Comment:
NIT: I would favor `FungibleStakedSui` or `ActivatedStakedSui` to keep the naming schema the same between this object and `StakedSui`.